### DISCUSSION TITLE
Could not poll the database: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known

### DISCUSSION THREAD

**Discussion by btk-42** [firefly-iii/firefly-iii#6103] [Firefly III]
Hello, 

I saw some people had the same issue but there were different solution for it. I tried some but didn't work. 
I'm new with Docker and Linux and I'm still learning it. 

All information in the attachment.

[_ff3-app-1_logs.txt](https://github.com/firefly-iii/firefly-iii/files/8774123/_ff3-app-1_logs.txt)
[env.txt](https://github.com/firefly-iii/firefly-iii/files/8774179/env.txt)
[docker-compose.txt](https://github.com/firefly-iii/firefly-iii/files/8774181/docker-compose.txt)
[_ff3-db-1_logs.txt](https://github.com/firefly-iii/firefly-iii/files/8774204/_ff3-db-1_logs.txt)

![image](https://user-images.githubusercontent.com/106232568/170353054-50ceab9f-7333-483a-92af-072324e9abfd.png)

**Comment by JC5** [comment]
Either change `DB_HOST=db` to `DB_HOST=fireflyiiidb` in your .env file, OR remove the line `hostname: fireflyiiidb` from your Docker Compose file.

That should fix it.

**Reply by btk-42** [reply]
Hi, 

Same result as before :/

**Reply by JC5** [reply]
Then I don't know. The error in your settings is easy to see. Your `.env` file says the database is at "db" but your `docker-compose.yml` file says it's called "fireflyiiidb". One of them has to change.

Make sure you remove the volumes and containers and restart them all.

**Reply by btk-42** [reply]
I saw on portainer that my db container doesn't get ip address. I made new files exactly the same as in guide and it is working now :) 

Sorry for wasting your time

**Reply by JC5** [reply]
Not a problem, glad it works!
